### PR TITLE
Added contributing guidelines and link from readme

### LINKED ISSUE #2133: Contribution guidelines are hard to find
Before I submitted a pull request I was looking for some contribution guidelines but I couldn't find any. Well, they do exist, I just couldn't find them. Maybe add a link in README or something?

http://nitlanguage.org/internal/patches.html

Just a thought.

### PR DESCRIPTION
**Pull request by R4PaSs**
As discussed in #2133, contribution guidelines are hard to find.

This PR is an attempt to ease the process by giving newcomer friendly advices and some more insight on the contribution process for Nit.

@tonygaetani, do you think a document like this one would have been helpful for you ? Is there anything you would change to make it friendlier/clearer ?

Close #2133

### CODE REVIEW

**Comment by jcbrinfo** [comment]
There is a checklist for new contributors here: http://nitlanguage.org/internal/

In fact, this is this page that links to http://nitlanguage.org/internal/patches

**Comment by jcbrinfo** [comment]
Not a fan of copy-pasting what is already in the wiki. This is why we invented hyperlinks. :wink:

**Comment by privat** [comment]
I agree with @jcbrinfo . Just use hyper-links luke. We do not want to maintain two places.

In fact, a `CONTRIBUTING.md` with just a simple sentence and links should be enough;
* Information entry point is at a standard place
* Full information is not duplicated

**Comment by tonygaetani** [comment]
@R4PaSs The first thing I looked for was a CONTRIBUTING.md, so this definitely would have been helpful to me! 

For what it's worth I agree about the hyperlinks and not having to maintain the same content in multiple places. Just having a CONTRIBUTING.md with a link is great.

**Comment by R4PaSs** [comment]
Reroll: Replaced the guidelines paragraph by a link to the website

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+All the contributions to the Nit language, its tools and libraries, are done through Github Pull-Requests.
+
+All the information on our contribution policy is available on our [website](http://nitlanguage.org/internal/).
+We encourage you to take a look at the sections available here for a walthrough of what is required to submit a new PR.
+
+### I'm a beginner and I'd like to program in Nit, where do I start ?
+
+If you are looking for inspiration to begin programming in Nit, we encourage you to take a look at our issues. Some of them are marked with the `newcomer` tag.
+These issues are aimed at beginners with the language, and are a way to encourage people with no experience with Nit to engage in easy, fun tasks.
```
Comment (jcbrinfo) [inline CONTRIBUTING.md]: Already stated by the “Get Involved” page. If the wiki not clear enough, you should update it instead of paraphrasing it IMHO.

**Comment by R4PaSs** [comment]
Reroll: remove the newcomer section in the CONTRIBUTING

**Pull request merged**